classes/message_processor.php

```diff
@@ -1,9 +1,12 @@
 <?php
 namespace tool_messagebroker;
 
+use coding_exception;
+use messagebrokerdatastore_standarddb\durable_dao;
```

Comment:
The processor should know nothing about the concrete implementations of the data stores.

Reply:
Updated the branch. Everything should be using `durable_dao_interface` now.